volbuild/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM meteorhacks/meteord:base
+MAINTAINER MeteorHacks Pvt Ltd.
+
+ONBUILD ENV METEORD_VOLBUILD=true
```

Comment:
Using this PR I find that `METEORD_VOLBUILD` environment in the container isn't set by this line. Instead requires you to pass it as an arg to `run`. Does it actually work for you?